src/main/io/ledstrip.c

```diff
@@ -917,7 +917,7 @@ static void applyRainbowLayer(bool updateNow, timeUs_t *timer)
             rainbowLedIndex++;
         }
     }
-    offset++;
+    offset+=2;
```

Comment:
Please explain this change (probably 2 byte index)
```suggestion
    offset += 2;
```

Reply:
It's just doubles the effect speed, it was requested on Discord and I didn't want to increase LEDSTRIP task speed so instead I'm adding double of the HUE difference as before on each iteration. Like this we actually skipping half of the colors but it's not visible to human eyes.